components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -24,20 +27,31 @@
 
 /* kBraveWalletKeyrings structure
  *
- * { "default":
+ * {
+ *   "hardware":  {
+ *     "Ledger12445": {
+ *        "account_metas": {
+ *            "0xEA04...CC8Acc": {
+ *              "account_name": "Ledger 1",
+ *              "derivation_path": "m/44'/60'/1'/0/0",
+ *           },
+ *           "0x264Ef...6b8F1": {
+ *              "account_name": "Ledger 2",
+ *              "derivation_path": "m/44'/60'/2'/0/0",
+ *            }
+ *        },
+ *        device_name: "Ledger 123"
```

Comment:
I don't see interface to update it in mojom. When you are about implement this, make sure the API takes hardware_id as args

Reply:
as discussed created new issue for this https://github.com/brave/brave-browser/issues/18165